daemon/src/model/cfd.rs

```diff
@@ -2371,6 +2390,212 @@ mod tests {
         assert_eq!(funding_fee.rate, funding_rate);
     }
 
+    #[test]
+    fn given_collab_settlement_then_cannot_start_rollover() {
+        let quantity = Usd::new(dec!(10));
+        let leverage = Leverage::new(2).unwrap();
+        let opening_price = Price::new(dec!(10000)).unwrap();
```

Comment:
These values are not important for the test right? Would be good to hide them then! Would `Cfd::taker_long().dummy_open()` be sufficient?

Reply:
For tests that actually do a collaborative settlement (for either role) they are important because otherwise we run into different errors. For other tests I can try to remove them.